lib/rdoc/generator/template/darkfish/css/rdoc.css

```diff
@@ -96,6 +96,8 @@ main .anchor-link:target {
 a {
   color: var(--link-color);
   transition: color 0.3s ease;
+  text-decoration: underline;
+  text-underline-offset: 2px; /* Make sure it doesn't overlap with underscores in a method name. */
```

Comment:
[note] Why not more spaces like 3px? If it becomes 3px, an inline code with a link looks like (this example is on the LEGAL page): I'm a bit concerned that the underline sticks to the bottom line of the inline code box.
<img width="535" alt="image" src="https://github.com/user-attachments/assets/cf0ddd08-22e2-4317-99b3-067092848fc3" />

In the case of 2px:
<img width="522" alt="image" src="https://github.com/user-attachments/assets/eec8aa36-6ed0-4063-9c19-30639038e470" />

But if people are in favor of 3px or more, I'm glad to change it. 😃

Reply:
I prefer 3px to 2px but I'm not good at visual design...

Others should decide it...

Reply:
Not a design expert either, but I find the 3px version better.